pkg/loki/config_wrapper.go

```diff
@@ -619,14 +619,32 @@ func applyEmbeddedCacheConfig(r *ConfigWrapper) {
 
 	indexStatsCacheConfig := r.QueryRange.StatsCacheConfig.CacheConfig
 	if !cache.IsCacheConfigured(indexStatsCacheConfig) {
+		prefix := indexStatsCacheConfig.Prefix
```

Comment:
we set it during [flag registration](https://github.com/grafana/loki/blob/87061625cb8cc3ab7b2095fdd86daa42c3a9644f/pkg/storage/chunk/cache/cache.go#L59)

had to add this line since were previously causing a collision by not retaining it (when copying results cache config)

Reply:
Oh nice! Thanks